src/VR.js

```diff
@@ -610,37 +610,54 @@ class FakeVRDisplay extends VRDisplay {
 const createVRDisplay = () => new FakeVRDisplay();
 
 const controllerIDs = {
-  oculusVRIDLeft: 'Oculus Touch (Left)',
-  oculusVRIDRight: 'Oculus Touch (Right)',
-  oculusMobileVRIDLeft: 'Oculus Touch (Left)',
-  oculusMobileVRIDRight: 'Oculus Touch (Right)',
-  openVRID: 'OpenVR Gamepad',
-  openVRTrackerID: 'OpenVR Tracker'
+  OculusVRLeft: 'Oculus Touch (Left)',
+  OculusVRRight: 'Oculus Touch (Right)',
+  OculusQuestLeft: 'Oculus Touch (Left)',
+  OculusQuestRight: 'Oculus Touch (Right)',
+  OculusGo: 'Oculus Go',
+  GearVR: 'Gear VR',
+  OpenVR: 'OpenVR Gamepad',
+  OpenVRTrackerID: 'Tracker'
 };
 
+function getControllerID(vrDisplay, hand) {
+  if (!vrDisplay) { return; }
```

Comment:
We always have a `vrDisplay` when requesting a controller ID so there's probably no reason to check this, right?

Reply:
done